smarts/core/sumo_road_network.py

```diff
@@ -796,6 +824,8 @@ def road_with_point(self, point: Point) -> RoadMap.Road:
         for nl, dist in self.nearest_lanes(point, radius):
             if dist < 0.5 * nl._width + 1e-1:
                 return nl.road
+        else:
+            print(f"Point is off at {point}")
```

Comment:
Is it possible for the `self.nearest_lanes()` to ever return an empty list?

Reply:
This keeps sneaking back in, I thought I removed this debugging.

Reply:
Yes, it is possible for nearest lanes to be empty if a vehicle strays far enough from the road.

In this case the `else` was entirely unnecessary because there is a `return` rather than a `break`...